examples/mpu9250-kalman-pitch.rs.outdated

```diff
@@ -2,38 +2,32 @@
 
 #![deny(unsafe_code)]
 #![deny(warnings)]
-#![feature(proc_macro)]
+#![no_main]
```

Comment:
same for this one

Reply:
Imu struct mpu9250 crate is private, so I don't know how to create _static_ init value for rtfm.

Reply:
OK